src/components/Contact/ContactLinks.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import styled from "styled-components";
+import devices from '../../utility/MediaQueries.js';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const LinksContainer = styled.div`
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    max-width: 100%;
+    min-width: 20rem;
+    ${devices.small`
+        justify-content: space-evenly;
+    `}
+    a {
```

Comment:
Favor creating a styled-component for the links instead of throwing styles in the parent.